sdk/storage/azure-storage-blob/tests/test_append_blob.py

```diff
@@ -1326,14 +1517,24 @@ def test_copy_sealed_blob_with_seal_blob_will_get_a_sealed_blob(self, storage_ac
         copied_blob3.start_copy_from_url(blob.url, seal_destination_blob=False)
         prop = copied_blob3.get_blob_properties()
 
-        self.assertIsNone(prop.is_append_blob_sealed)
+        assert prop.is_append_blob_sealed is None
         copied_blob3.append_block("abc")
 
     @BlobPreparer()
-    def test_create_append_blob_with_immutability_policy(self, versioned_storage_account_name, versioned_storage_account_key, storage_resource_group_name):
+    @recorded_by_proxy
+    def test_create_append_blob_with_immutability_policy(self, **kwargs):
+        versioned_storage_account_name = kwargs.pop("versioned_storage_account_name")
+        versioned_storage_account_key = kwargs.pop("versioned_storage_account_key")
+        storage_resource_group_name = kwargs.pop("storage_resource_group_name")
+
+        variables = kwargs.pop("variables")
+        if self.is_live:
+            expiry_time = datetime.utcnow() + timedelta(seconds=10)
+            variables = {"expiry_time": expiry_time.isoformat()}
+
         bsc = BlobServiceClient(self.account_url(versioned_storage_account_name, "blob"), versioned_storage_account_key, max_block_size=4 * 1024)
         self._setup(bsc)
-
+    
```

Comment:
Is this a tab? I'm surprised PyLint hasn't kicked you to the curb for these ones, as compared to something like line 1509 which is just a newline

Reply:
It is a tab, strange. Yeah also surprised PyLint didn't yell. Will fix them up.